modules/indexing/src/main/java/org/apache/ignite/internal/processors/query/h2/DmlStatementsProcessor.java

```diff
@@ -95,62 +89,14 @@ public ModifyingEntryProcessor(Object val, IgniteInClosure<MutableEntry<Object,
         }
     }
 
-    /** Dummy anonymous class to advance RMV anonymous value to 5. */
-    private static final Runnable DUMMY_1 = new Runnable() {
-        @Override public void run() {
-            // No-op.
-        }
-    };
-
-    /** Dummy anonymous class to advance RMV anonymous value to 5. */
-    private static final Runnable DUMMY_2 = new Runnable() {
-        @Override public void run() {
-            // No-op.
-        }
-    };
-
-    /** Dummy anonymous class to advance RMV anonymous value to 5. */
-    private static final Runnable DUMMY_3 = new Runnable() {
-        @Override public void run() {
-            // No-op.
-        }
-    };
-
-    /** Remove updater for compatibility with < 2.7.0. Must not be moved around to keep at anonymous position 4. */
-    private static final IgniteInClosure<MutableEntry<Object, Object>> RMV_OLD =
-        new IgniteInClosure<MutableEntry<Object, Object>>() {
+    /** Remove updater. */
+    public static final IgniteInClosure<MutableEntry<Object, Object>> RMV =
```

Comment:
Let's make this class named and static to overcome future issues with RU that was solved with the `DUMMY` runnables.